polars/polars-ops/src/chunked_array/datetime/replace_time_zone.rs

```diff
@@ -0,0 +1,24 @@
+use polars_arrow::kernels::replace_time_zone as replace_time_zone_kernel;
+use polars_core::prelude::*;
+
+pub fn replace_time_zone(
+    series: &DatetimeChunked,
```

Comment:
Can you name this `ca`. `series` is actually for `Series` where we don't know the concrete type.

Reply:
sorry about that, I should've noticed - have updated!